css/dateRangePicker.hbs

```diff
@@ -0,0 +1,14 @@
+<label>Template type:</label>
+<select ng-model="dateRangeTemplateType" ng-options="option for option in dateRangeTemplateTypes"></select>
```

Comment:
I know it's not new, but is it possible to change the select for a md-select?
So the styleguide follows the styleguide ^^ *inception*